api.yml

```diff
@@ -1,159 +1,62 @@
 openapi: 3.0.0
-# Added by API Auto Mocking Plugin
-servers:
-  - description: SwaggerHub API Auto Mocking
-    url: https://virtserver.swaggerhub.com/danubetech/credential-issuer/0.0.1
 info:
-  description: This is an open API specification for issuing digital credentials based on standards such as the Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/), JWT (https://tools.ietf.org/html/rfc7519), and Open Badges (https://openbadges.org/). Implementations of this API may differ in terms of which data formats or proof formats they do or do not support. Also, implementations may choose to not support certain parts of this API (e.g. optional parts include changing and retrieving the status of a credential, or refreshing a credential).
-  version: "0.0.1"
-  title: Credential Issuer API
+  description: This is an open HTTP API specification for issuing Verifiable Credentials based on the W3C Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/). Implementations of this API may differ in terms of which representation formats or proof formats they do or do not support.
+  version: "0.0.2"
+  title: Verifiable Credential Issuer HTTP API
   contact:
-    email: markus@danubetech.com
+    email: george.aristy@securekey.com
   license:
     name: All documents in this Repository are licensed by contributors under the [W3C Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
-tags:
-  - name: internal
-    description: Internal APIs facing the Issuer.
-  - name: public
-    description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials/sign:
     post:
-      tags:
-        - internal
-      summary: Issues a credential and returns it in the response body.
-      operationId: issueCredential
-      description: Issues a credential and returns it in the response body. Support of this part of the API is REQUIRED for implementations.
-      parameters:
-        - in: query
-          required: false
-          name: templateReference
-          schema:
-            type: string
-          description: Optional internal reference to a credential template. If this is present, it may be used to automatically pre-populate certain fields in the request body.
-          example: 67466513-67a0-4812-8f26-995dd78c5aa6
-        - in: query
-          required: false
-          name: subjectReference
-          schema:
-            type: string
-          description: Optional internal reference to the subject for which to issue a credential. If this is present, it may be used to pre-populate certain fields in the request body (e.g. "claims" and/or "subject").
-          example: 73b32334-81f3-4f4c-911e-710c7aadd4d8
+      summary: Constructs a credential from a template, attaches proof, and returns it in the response body.
```

Comment:
-1 for "constructing a credential from a template" (if template means "something that allows you to plug values into it") -- if this is a signing API, you should pass in the unsigned thing, and the API signs it.